### ISSUE TITLE
Accessibility audit

### ISSUE THREAD

**Issue by kettanaito** [open-draft/presenteer#8]
Presenteer should provide a tool to run automated accessibility tests against any given Presenteer presentation.

**Such accessibility tests may assert:**

1. Foreground/background contrast ratio.
1. Font size. All people should see what's written on a slide, even when sitting further away. 
1. Colors contrast.
1. Transition speed. Too fast transitions may cause dizziness.

## API

```bash
$ presenteer audit ./MasterSlide.jsx
```

The output is similar to any other automated test software output (mocha, cucumber), and represents a list of checks, each having its own pass/failed status.
